pymc3/distributions/simulator.py

```diff
@@ -0,0 +1,69 @@
+import theano.tensor as tt
+import numpy as np
+from .distribution import NoDistribution
+
+__all__ = ['Simulator']
+
+
+class Simulator(NoDistribution):
+   
+    def __init__(self, function, *args, **kwargs):
+        """
+        This class stores a function defined by the user in python language.
+        
+        function : function
+            Simulation function defined by the user.
+        *args and **kwargs : 
+            Arguments and keywords arguments that the function takes.
+        """
+
+        self.function = function
+        observed = self.data
+        super(
```

Comment:
With python3 you can just call `super()`

Reply:
@aloctavodia